float32 enemy_dist
float32 enemy_yaw
float32 enemy_pitch
